lib/cmd/fh3/keys/ssh/add.js

```diff
@@ -0,0 +1,52 @@
+/* globals i18n */
+var common = require('../../../../common.js');
+var fhreq = require("../../../../utils/request");
+var fs = require('fs');
+var fhc = require("../../../../fhc");
+
+module.exports = {
+  'desc' : i18n._('Add a SSH Key Management'),
```

Comment:
Ah i see. In the Studio it's called `SSH Key Management` because it is a whole section in the Studio for multiple actions (list of keys, add, remove and edit them). But the fhc commands are all just single action. I think it's better to not use `Management` in the description here.